src/screens/Home.tsx

```diff
@@ -20,46 +20,131 @@ const Home = (): JSX.Element => {
     //const url = ""
     //const textJson = loadParseJson(url);
     //hardcoded json file for now until can use fetch to get texts from server
-    const textJson = {'title': "Text Title", 'content': "This is a paragraph of text, I'm making this longer to see what happens when it reaches the edge of the phone screen, spoiler it wraps. \n\nThis is a second paragraph.", 'audio_file_id':"1"}
+    const textJson = {'title': "Neoclassicism and Early Romanticism In Britain", 'content': "British tourists and artists in Italy were the leading supporters of early Neoclassiscism, partly because of the burgeoning taste for revival styles at home. Nonetheless, the British interest in Classical revival styles was inflected slightly differently from Roman Neoclassiscism. While Roman Neoclassiscism looked to the past in order to revive a sense of moral and civic virtue, many later eighteenth-century British artists harnessed the concept of civic virtue to patriotism to create more Romantic works of art dedicated specifically to the British. In conclusion, Neoclassicism in Britain was a significant artistic and architectural movement that celebrated the timeless beauty of classical antiquity, contributed to the Enlightenment's intellectual climate, and left an enduring mark on British culture and aesthetics. It remains an important chapter in the history of British art, architecture, and literature.", 'audio_file_id':"1"}
     setTitle(textJson.title);
     setContent(textJson.content);
   }
 
   return (
-    <View> 
-      <Text style={styles.titleText}>{title}</Text>
-      <Text style={styles.baseText}>{content}{'\n'}</Text>
-      <Button
-        title= 'Display Text'
-        onPress={displayText}
-       />
-      <Button
-        title='Get audio'
-        onPress={textToAudio}
-      />
-      <AudioPlayer/>
+    <View style={styles.homeBackground}>
+      <ScrollView>
```

Comment:
If you change ScrollView to contain only these three, you can get the floating bar at the bottom:
```
<ScrollView>
    <View style={styles.topBarContainer}>
            ...    
    </View>
    <Text style={styles.titleText}>{title}</Text>
    <Text style={styles.baseText}>{content}{'\n'}</Text>
</ScrollView>
```